helsingborg-stad/app-mitt-helsingborg
# feat: show already uploaded attachments for completion

## Explain the changes you’ve made
Added the ability to show already uploaded attachments in the `BulletList` component

## Explain why these changes are made
During completion and random check, we would like to show attachments thats already been uploaded, hence doing this change.

## Explain your solution
Followed the same pattern as we do with requested completions

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Make sure you have a case for completions or random check
4. make sure you have an array of strings (file names) in path `details.completions.attachmentUploaded`
5. Make sure that the `BulletList` component in formbuilder has a bullet with the string `#completionsUploaded`  

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
